tests/test_functional_sqlite.py

```diff
@@ -84,15 +229,90 @@ def test_all_indexes(self, conn, TableWithIndexes, setup_tables):
 
         indexes_after_drop = conn.execute(text("""SELECT * FROM "geometry_columns";""")).fetchall()
         tables_after_drop = conn.execute(
-            text("SELECT name FROM sqlite_master WHERE type ='table' AND name NOT LIKE 'sqlite_%';")
+            text(
+                "SELECT name FROM sqlite_master WHERE type = 'table' AND name NOT LIKE 'sqlite_%';"
+            )
         ).fetchall()
 
         assert indexes_after_drop == []
         assert [table for table in tables_after_drop if "table_with_indexes" in table.name] == []
 
 
 class TestInsertionORM:
-    pass
+    @pytest.fixture
+    def LocalPoint(self, base):
+        class LocalPoint(base):
+            __tablename__ = "local_point"
+            id = Column(Integer, primary_key=True)
+            geom = Column(
+                TransformedGeometry(
+                    db_srid=2154, app_srid=4326, geometry_type="POINT", management=False
+                )
+            )
+
+        return LocalPoint
+
+    def test_transform(self, session, conn, LocalPoint, setup_tables, dialect_name):
+        if dialect_name == "sqlite-gpkg":
+            # For GeoPackage we have to create the 'spatial_sys_ref' table to be able to use
+            # the ST_Transform. It can be created using InitSpatialMetaData() but it also creates
+            # the 'geometry_columns' table, which is useless. So here we create the table manually
+            # with only the required SRS IDs.
+            # TODO: Should we move this to a public function so people can use it easily?
+            conn.execute(
+                text(
+                    """CREATE TABLE IF NOT EXISTS spatial_ref_sys (
+                      srid       INTEGER NOT NULL PRIMARY KEY,
+                      auth_name  VARCHAR(256),
+                      auth_srid  INTEGER,
+                      srtext     VARCHAR(2048),
+                      proj4text  VARCHAR(2048)
+                    );"""
+                )
+            )
+            conn.execute(
+                text(
+                    """INSERT INTO spatial_ref_sys
+                    SELECT
+                      srs_id AS srid,
+                      organization AS auth_name,
+                      organization_coordsys_id AS auth_srid,
+                      definition AS srtext,
+                      NULL
+                    FROM gpkg_spatial_ref_sys AS A
+                    WHERE NOT EXISTS (SELECT srid FROM spatial_ref_sys WHERE srs_id = A.srs_id);"""
+                )
+            )
```

Comment:
I was not able to make the `ST_Transform` function without it. IMO it's something that should be fixed on the Spatialite side but until then we could create a function to make it easier for users.

Reply:
I think such a function would be required for our application; we use `ST_Transform` at some points

Reply:
Ok, I will add it then.